hnsendpoint.go

```diff
@@ -6,6 +6,8 @@ import (
 
 // HNSEndpoint represents a network endpoint in HNS
 type HNSEndpoint = hns.HNSEndpoint
+// Namespace represents a Compartment.
+type Namespace = hns.Namespace
```

Comment:
What uses this? I dont quite understand why this export is needed for any V1 code?